pytorch_lightning/metrics/sequence.py

```diff
@@ -0,0 +1,51 @@
+from typing import List, Sequence
+
+import torch
+
+from pytorch_lightning.metrics.functional.sequence import bleu_score
+from pytorch_lightning.metrics.metric import Metric
+
+
+class BLEUScore(Metric):
+    """
+    Calculate BLEU score of machine translated text with one or more references.
+
+    Example:
+
+        >>> translate_corpus = ['the cat is on the mat'.split()]
```

Comment:
@awaelchli can we also add here a conditional skip in case that `torchtext` is not installed?

Reply:
No, for pytest doctesting this is not possible. 
Conditional skipping can only be done with sphinx doctests. 
I suggest to install torchtext in the doctest CI job or rewrite the test without torchtext

Reply:
well the purpose of these test it to check if the package can run without extras, so install extras extra goes against this test :]
so just add this module to ignore for the particular test... (as sklearns already is)

Reply:
Should I add `--ignore=pytorch_lightning/metrics/sequence.py`?